cni/network/network.go

```diff
@@ -1013,8 +1039,8 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 
 		if !nwCfg.MultiTenancy {
 			// Call into IPAM plugin to release the endpoint's addresses.
+			logAndSendEvent(plugin, fmt.Sprintf("Releasing ips:%+v", epInfo.IPAddresses))
```

Comment:
i would prefer not changing this. anyway it will add one more extra log which is fine imo.. it will give more clarity on ip has been released and what not